modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -454,24 +476,37 @@ private void performCommitWrite(UUID txId, Set<RowId> pendingRowIds, HybridTimes
      */
     private void performAbortWrite(UUID txId, Set<RowId> pendingRowIds, @Nullable List<Integer> indexIds) {
```

Comment:
I believe the second usage of this method only passes `Set.of(rowId)`, which means that the method is redundant